GNUmakefile

```diff
@@ -5,12 +5,13 @@ WEBSITE_REPO=github.com/hashicorp/terraform-website
 MODULE   = $(shell env GO111MODULE=on go list -m)
 VERSION ?= $(shell git describe --tags --always --dirty --match=v* 2> /dev/null || \
 			cat $(CURDIR)/.version 2> /dev/null || echo v0)
+VERSION_NUMBER = $(shell echo $(VERSION) | sed 's/v//' | sed 's/-dirty//')
```

Comment:
Probably don't need to do `| sed 's/-dirty//'` if you removed the `--dirty` flag in the line above 🤔

Reply:
The dirty thing feels ok to me

Reply:
I think that the original `VERSION` should be `GIT_VERSION` and this new one should be `VERSION`

Reply:
Will change that! 👍